Github Issue Reader/Views/IssuesCollectionVC.swift

```diff
@@ -64,9 +62,8 @@ class IssuesCollectionVC: UICollectionViewController, UICollectionViewDelegateFl
         return 1
     }
     
-//    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-//        return CGSize.init(width: view.frame.size.width, height: 250)
-//    }
-    
-    // collectionView data source protocols - common pattern
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let fullWidth = collectionView.bounds.size.width - collectionView.layoutMargins.left - collectionView.layoutMargins.right
+        return CGSize(width: fullWidth, height: 100)
```

Comment:
you can make a constant for the height  (e.g. `let cellHeight: CGFloat = 100`) - that'll make it easier for someone later on to change (this really should be EIGHTY SEVEN POINTS AND NO MORE!!1!!), and if anything else needs to be 100 points tall to match the cell height